sphericart-torch/CMakeLists.txt

```
@@ -19,6 +19,7 @@ check_language(CUDA)
 if(CMAKE_CUDA_COMPILER)
     enable_language(CUDA)
     set(CUDA_USE_STATIC_CUDA_RUNTIME OFF CACHE BOOL "" FORCE)
+    add_compile_definitions(CUDA_AVAILABLE)
```

this should be 
```suggestion
    target_compile_definitions(sphericart_torch PRIVATE CUDA_AVAILABLE)
```
or 
```suggestion
    target_compile_definitions(sphericart_torch PUBLIC CUDA_AVAILABLE)
```

`PRIVATE` if this is only set when compiling sphericart, `PUBLIC` if this is to be defined when using pre-compiled sphericart in another project.

---

I've committed the PRIVATE statement.